L1Trigger/VertexFinder/python/VertexProducer_cff.py

```diff
@@ -51,21 +55,31 @@
         # DBSCAN min density tracks
         DBSCANMinDensityTracks = cms.uint32(2),
         # Minimum pt of tracks used to create vertex [GeV]
-        VxMinTrackPt = cms.double(2.0),
+        VxMinTrackPt = cms.double(1.9),
         # Maximum pt of tracks used to create vertex [GeV]
-        VxMaxTrackPt = cms.double(127.0),
+        VxMaxTrackPt = cms.double(512.0),
```

Comment:
Previously talking to Alexx he had said to use 127 here

Reply:
Changed, will also retrain models with this cut as it will impact performance